handbook/engineering/core-application/operational-rotation.md

```diff
@@ -0,0 +1,23 @@
+# Operational rotation - how do we provide support to other teams? 
+
+Our team works in biweekly iterations applying SCRUM principles. Each sprint has defined goals and fixed scope that the team has committed to deliver. We also need to support our customers and reply to different types of requests from other teams (for example, answering questions, reviewing and merging pull requests, or performing necessary maintenance tasks). While all of these (outside of the sprint) tasks are essential and could be urgent, we consider them interruption work because they are not part of currently planned sprint commitments. 
+
+We want to **focus on the sprint and have the flow** without constantly switching contexts. We also want to be **predictable with our commitments**. At the same time, we acknowledge that **there will always be unplanned work that needs our urgent attention**. 
+
+We are establishing the **operational rotation** to predictably deliver sprint goals and provide the necessary support to other teams. Each iteration, a **single engineer** from the team will play a goalie role and take responsibility for all open requests for assistance. It's a **rotation-based** assignment with a defined [schedule](https://sourcegraph.app.opsgenie.com/settings/schedule/detail/b553cefc-2466-4ad2-ad0c-66937c790bbf). All requests for help that fall into this category will be worked on within this engineer's **regular office hours**. We are tracking these requests in a separate [Core Application Support Github project](https://github.com/orgs/sourcegraph/projects/153#card-65409816) to keep us organized and transparent.
+
+**_IMPORTANT: Operational rotation and on-call rotation are two separate roles with different responsibilities._**
```

Comment:
Should we mention the responsibilities of the on call engineer?

Reply:
yes - we need to define this as a separate working agreement. Actually, this is something I'd like to delegate to somebody from the team to own, agree with the team, and document. Planning to start a conversation about this next week